languages.toml

```diff
@@ -883,6 +883,10 @@ injection-regex = "(shell|bash|zsh|sh)"
 file-types = [
   "sh",
   "bash",
+  "ash",
```

Comment:
On second thought, does anybody still use `ash` nowadays? Even Android deprecated it in favor of MKSH

Reply:
Seems fine to include it if no other language is using it